.circleci/config.yml

```
@@ -37,7 +37,8 @@ jobs:
             git clone --depth 1 https://github.com/plotly/dash.git
             cd dash && pip install -e .[testing] --quiet && cd ..
             export PATH=$PATH:/home/circleci/.local/bin/
-            pytest tests/integration/
+            pytest tests/integration/ --headless --nopercyfinalize
```

```suggestion
            pytest --headless --nopercyfinalize tests/integration/
```
not entirely sure if this matters, but normally the flags come before the other args.